xcube/core/normalize.py

```diff
@@ -59,7 +59,12 @@ def normalize_dataset(ds: xr.Dataset) -> xr.Dataset:
 
     # xcube viewer currently requires decreasing latitude co-ordinates, so
     # we invert them here if necessary.
-    ds = _ensure_lat_decreasing(ds)
+
+    # TODO: commented out as _ensure_lat_decreasing() produces
+    #   chunks in "lat" that have the smallest chunk first. This will
+    #   fail when writing to Zarr. (Hack for #347)
+    #   See also https://github.com/pydata/xarray/issues/2300
+    # ds = _ensure_lat_decreasing(ds)
```

Comment:
Why only commented out? Why not fully removed? And the whole method along with it?

Reply:
Maybe because it's only intended as a temporary hack? Not sure...